### PR TITLE
Add env to logging

### PR DESCRIPTION
**Pull request by d-sparks**
We want this for routing these to signalfx.

Relates to: https://github.com/Clever/salt/pull/903

### CODE REVIEW

File: lib/metrics.coffee [hunk 1]
```diff
@@ -1,13 +1,15 @@
 _ = require 'underscore'
 kayvee = require 'kayvee'
 
+env = process.env.ENV or 'development'
```
Comment (kvigen) [inline lib/metrics.coffee]: same question about staging versus development as the other PR

File: lib/metrics.coffee [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
+
 # log_memory_usage logs HeapUsed, HeapTotal, and RSS in the kayvee format
 log_memory_usage = (source) ->
     mem = process.memoryUsage()
-    console.error kayvee.formatLog(source, kayvee.INFO, 'HeapUsed', {type: "gauge", value: mem.heapUsed})
-    console.error kayvee.formatLog(source, kayvee.INFO, 'HeapTotal', {type: "gauge", value: mem.heapTotal})
-    console.error kayvee.formatLog(source, kayvee.INFO, 'RSS', {type: "gauge", value: mem.rss})
-
+    type = "gauge"
+    console.error kayvee.formatLogsource, kayvee.INFO, 'HeapUsed', {type, env, value: mem.heapUsed}
```
Comment (kvigen) [inline lib/metrics.coffee]: s/formatLogsource/formatLog source

File: lib/metrics.coffee [hunk 3]
```diff
@@ -40,7 +44,10 @@ start_pause_detector = (source, sleep_time_ms, pause_threshold_ms) ->
   setInterval pause_fn, sleep_time_ms
 
 log_pauses = (source) ->
-  console.error kayvee.formatLog(source, kayvee.INFO, 'PauseMetric', { type: "gauge", value : module.exports._last_period_pause_ms})
+  console.error kayvee.formatLog source, kayvee.INFO, 'PauseMetric',
```
Comment (kvigen) [inline lib/metrics.coffee]: does it make sense to add a helper function that automatically adds env to all log messages or is that just unneeded indirection?

Comment (d-sparks) [inline lib/metrics.coffee]: I started to do that, but it seemed too complicated for how short this script is.

Comment (kvigen) [inline lib/metrics.coffee]: fair enough

**Comment by kvigen** [comment]
lgtm

**Pull request merged**